woocommerce/woocommerce
# Update states.php -> Add Morocco regions

### Submission Review Guidelines:

-   I have followed the [WooCommerce Contributing Guidelines](https://github.com/woocommerce/woocommerce/blob/trunk/.github/CONTRIBUTING.md) and the [WordPress Coding Standards](https://make.wordpress.org/core/handbook/best-practices/coding-standards/).
-   I have checked to ensure there aren't other open [Pull Requests](https://github.com/woocommerce/woocommerce/pulls) for the same update/change.
-   I have reviewed my code for [security best practices](https://developer.wordpress.org/apis/security/).
-   Following the above guidelines will result in quick merges and clear and detailed feedback when appropriate.



### Changes proposed in this Pull Request:





Add regions for Morocco according to CLDR 

https://github.com/unicode-org/cldr/blob/f800890ea86482c2eb5f73224897f4a8bc0b653a/common/subdivisions/en.xml#L3207

See https://airtable.com/appra4zMRgiGscWB7/tblqKcO8HXWoSFKUR/viwVKXWZNUe7840Lb/recBOhxl2CFVmOIGl?blocks=hide



### How to test the changes in this Pull Request:



Using the [WooCommerce Testing Instructions Guide](https://github.com/woocommerce/woocommerce/wiki/Writing-high-quality-testing-instructions), include your detailed testing instructions:

1. Use Shipping Settings to test, `/wp-admin/admin.php?page=wc-settings&tab=shipping&zone_id=new`
2. Use the Region Picker to search for Morocco and ensure the results show the country and its regions. Note, the Region picker is new, so if you are testing this locally instead of on JN, be sure to run `pnpm build` from `plugins/woocommerce-admin`.



### Changelog entry




<details>

#### Significance




#### Type




#### Message 

Add Moroccan regions to states.php

#### Comment 

</details>
